lib/charms/operator_libs_linux/v0/juju_systemd_notices.py

```diff
@@ -115,8 +116,8 @@ def _on_slurmd_stopped(self, _: ServiceStoppedEvent) -> None:
 
 _logger = logging.getLogger(__name__)
 _juju_unit = None
+_observed_services = {}
```

Comment:
Would `_service_aliases` be more apt?

Reply:
I think `_observed_services` is a bit more clear because this map contains both the service name and alias as opposed to just one or the other.

If `_observed_services` was just a list of the aliases, then I would agree that `_service_aliases` would be a better name. However I think `_observed_services` is fine here since it's serving as a holder for both the service names and aliases.